plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should add a test that this file is well-formed, like the plugin XML tests in some of the other packages